<launch>
        <node pkg="icr" name="model_server" type="model_server" output="screen" ns="model_server">
            <param name="model_directory" type="string" value="$(find icr)/models/urdf/"/>
            <param name="reference_frame_id" type="string" value="/fixed"/>
            <remap from="set_object" to="/grasp_server/set_object"/> 
            <param name="gazebo_prefix" type="string" value="/gazebo" /> 
       </node>
</launch>





